host/cli/volume.go

```diff
@@ -0,0 +1,210 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/flynn/flynn/host/volume"
+	"github.com/flynn/flynn/pkg/cluster"
+	"github.com/flynn/go-docopt"
+)
+
+func init() {
+	Register("volume", runVolume, `
+usage: flynn-host volume list
+       flynn-host volume create [--provider] HOST
```

Comment:
And it should have a default of `zfs` (I think you'll need an Options section for that).